### PR TITLE
Update mention of coverage config file to `pyproject.toml` in documentation

### PR DESCRIPTION
**Pull request by henningbra**
## Description

Checklist:

- [x] I've made sure that tests are updated accordingly (especially if adding or updating a template option)
- [x] I've updated the documentation or confirm that my change doesn't require any updates

### CODE REVIEW

**Review by foarsitter — CHANGES_REQUESTED** [review]
In my opinion your change is to briefly. It is that project.toml is the location where it is configured but the file is not dedicated to coverage.

**Comment by henningbra** [comment]
I'm new to contributing - learning. Just wanted to contribute to point out an error in the docs. Feel free to close it :-)

Allauth is a fantastic package for our Human Elephant Conflict mitigation system in a non profit organization.

**Comment by foarsitter** [comment]
Your contribution if more then welkom @henningbra! 

I would suggest something like this: 

```
The configuration for ``coverage`` can be found in `pyproject.toml`. You can find out more about `configuring`_ ``coverage``.
```

File: docs/testing.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ If you're running the project locally with Docker, use these commands instead: :
 
    At the root of the project folder, you will find the `pytest.ini` file. You can use this to customize_ the ``pytest`` to your liking.
 
-   There is also the `.coveragerc`. This is the configuration file for the ``coverage`` tool. You can find out more about `configuring`_ ``coverage``.
+   There is also the `pyproject.toml`. This is the configuration file for the ``coverage`` tool. You can find out more about `configuring`_ ``coverage``.
```
Comment (browniebroke) [inline docs/testing.rst:46]:
```suggestion
   The configuration for ``coverage`` can be found in ``pyproject.toml``. You can find out more about `configuring`_ ``coverage``.
```

**Review by browniebroke — APPROVED** [review]
Thanks for the fix and good point @foarsitter, I've applied your suggestion.

**Review by foarsitter — APPROVED** [review]

**Comment by henningbra** [comment]
I'm new. After the merge should I delete branch?

**Comment by foarsitter** [comment]
Yes, I should delete it or even delete the hole fork.

**Pull request merged**